google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/flexible/FileConfirmationDialog.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2016 The Android Open Source Project
+ * Copyright 2016 Google Inc. All Rights Reserved.
```

Comment:
I suspect it's because the last time this file was committed was last year.
Should be good now.